### PR TITLE
#168105120 Landing page

### PR DESCRIPTION
**Pull request by kennyedward**
#### What does this PR do?
This pull request creates a landing page for users

#### Description of Task to be completed
- creates user's landing page

#### How should this be manually tested?
N/A

#### Any Background Context You want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[#168105120](https://pivotaltracker.com/story/show/168105120)

#### Screenshots (if appropriate)
Screenshot 2019-09-04 at 9 46 32 AM
Screenshot 2019-09-04 at 9 46 40 AM

#### Questions:
N/A

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -27,4 +27,6 @@ const store = createStore(rootReducer, initialState, composedEnhancers);
 
 sagaMiddleware.run(rootSaga);
 
+// store.dispatch({ type: 'GET_CATEGORY', payload: { query: 'emotion' } });
```
Comment (kennyedward) [inline src/store/index.js]: remove commented code

**Review by iverenshaguy — APPROVED** [review]

**Comment by iverenshaguy** [comment]
LGTM. I would not want us to use camelCased class names though. So please adjust this

**Comment by iverenshaguy** [comment]
@kennyedward ~Do I merge then you implement my feedback later?~ @Nennyfills gave me a great suggestion about creating a demo branch with all your features into a demo branch for the demo. I won't merge this until the feedback is implemented then. Thanks

Feedback:
- The navbar is not properly aligned to the top and side of the page. 
- This back to top button should slide the page slowly not as quickly as we have in our current implementation. 
- The card content should determine the size of the card. Right now, all our cards are the same size. The PO doesn't want this. 
- The selection modal should show a loader when the user clicks on her category interset so that she has feedback that something is happening and is not wondering why her page isn't loading. 
- Reloading the page shouldn't bring out the modal again, the selected category should be stored for further use. You can use any of the available browser storages.
- I didn't get any results when I selected the first category in the modal, there should be a fallback set of articles to be rendered whenever the selected category returns no result so that we don't confuse the user
- Your test coverage should be over 95% please

**Review by iverenshaguy — CHANGES_REQUESTED** [review]
Would just request changes since we don't need to merge now

File: package.json [hunk 2]
```diff
@@ -36,11 +36,13 @@
   },
   "homepage": "https://github.com/andela/vali-ah-frontend#readme",
   "dependencies": {
+    "autoprefixer": "^9.6.1",
     "axios": "^0.19.0",
     "babel-polyfill": "^6.26.0",
     "debug": "^4.1.1",
     "dotenv": "^8.1.0",
     "express": "^4.17.1",
+    "favicons-webpack-plugin": "^1.0.1",
```
Comment (kennyedward) [inline package.json]: Place in dev dependencies

File: package.json [hunk 3]
```diff
@@ -106,6 +108,7 @@
     "require-context.macro": "^1.1.1",
     "sass-loader": "^7.3.1",
     "sass-rem": "^2.0.1",
+    "semantic-ui-css": "^2.4.1",
```
Comment (kennyedward) [inline package.json]: Place this under dependencies.

File: src/assets/scss/style.scss [hunk 4]
```diff
@@ -15,3 +16,39 @@ body {
...
+}
+
+.app-container {
+  width: 100%;
+  max-width: 81.25rem;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.show {
```
Comment (kennyedward) [inline src/assets/scss/style.scss:32]: Create a loader component that renders the framework's loader and overrides its styles.

File: src/components/ArticleCard/index.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+import viewsIcon from '../../assets/images/views.svg';
+
+const ArticleCard = ({
+  title = '',
+  summary = '',
+  category = ''
+}) => (
+  <a href="/#" className="card-wrapper">
+    <div className="card-wrapper__card">
```
Comment (kennyedward) [inline src/components/ArticleCard/index.jsx]:
```suggestion
    <div className="card">
```

**Pull request merged**